### DISCUSSION TITLE
2024-09-13 - 🗣️ Matching Tone with Intent - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#9150] [Chewing the Fat]
https://www.ssw.com.au/rules/match-tone-with-intent

In the past, have you tried to change your tone?
Were your successful?

**Comment by sethdaily** [comment]
I have done it many times - mostly when others remind me to. When I do it, it's very successful
The trouble for me is remembering!

**Comment by tiagov8** [comment]
I've done it sometimes, being sometimes successful. 

I guess in several scenarios we have a good tone naturally... when we are enthusiastic about whatever subject. But we should watch and try to have a good tone even when we aren't so happy about it.

I noticed that in the video examples one of the most important differences is Brady's facial expression. A smile 😀 helps too.

**Comment by MarcusIrmscherSSW** [comment]
Yes, It can be quite difficult when your tone and your emotional state are far apart. 😢

**Comment by tkapa** [comment]
I feel that I'm typically aware of how I sound. 

There's times where I've tried to be more upbeat, and I can normally achieve that for a period of time.

**Comment by SamBWagner** [comment]
I actively control my tone all the time. Best advice I can give is to always keep group calls/conversations light and playful as the cost of a conversation failing is higher.

**Comment by leahy268** [comment]
>In the past, have you tried to change your tone?

Yes, if I was really angry then I would usually modify my tone as much as I could.

>Were your successful?

Mostly.  Sometimes it isn't easy.

**Comment by calumjs** [comment]
>In the past, have you tried to change your tone?
>Were your successful?

I tried to say Chinese words with tones and I found it very difficult to speak and listen

**Reply by babakamyljanovssw** [reply]
This remembered me my Chinese lessons at the uni, where we whole class repeated after teacher how "a" sounds with 4 different tones

**Comment by lukecookssw** [comment]
I'm sure we've all been on the receiving end of the phrase "It's not what you say, it's how you say it" as children. Doesn't hurt to get a reminder every now and then :smile:

**Comment by joshbermanssw** [comment]
I try to be quite conscious of how what I say comes across to others - this in mind I have definitely, upon reflection, realised my tone may have been misleading and have been more aware in future interactions

**Comment by RobJT-wq** [comment]
I think it's important to consider tone of voice however I've always thought of it as having 2 tones, regular tone vs a customer service voice.

**Comment by TobyChurches** [comment]
There are plenty of occasions where I have tried to change my tone. Generally it starts off well then I gradually lose focus on how I am talking and focus more on what I am talking about. I think this is a valuable skill, but it is something that you have to gradually train yourself to do.

**Comment by Levijj22** [comment]
Having a wife is a constant reminder that tone matters

**Reply by ozdemir-mehmet** [reply]
Get a husband them, c'mon it's 2024!

**Comment by BenDunkerleySSW** [comment]
Changing tone is imperative when you want to be more convincing. It's very useful for me when mooting or debating.

**Comment by uly1** [comment]
This is a good thing to practice. it may feel diingenuine to start with but you can fake it till you make it and it becomes your natural way of communicating.

Also there's some evidence that physically smiling or having an open posture can cause you to feel a bit happier or more confident. so you can swap the usual cause and effect :)

**Comment by ozdemir-mehmet** [comment]
Tone matters that's why I listen to Tone Loc, who's old enough to know Tone Loc without googling it. 

But, back to the topic, I generally go with the flow.

**Comment by kikibianc** [comment]
Yes - I have a different personality and tone when speaking English vs Portuguese. 
I have also changed my tone in English for Adam to pay attention to what I say and be more engaging 😁

**Comment by jakebayliss** [comment]
Go through phases of activiely thinking about tone when speaking. Doesnt always stick but I think its something we should be always thinking about. Tone is sometimes more important than the words themselves

**Comment by awaltos** [comment]
I feel that I have developed this skill through years of customer service experience. In a previous life I was a mechanic and had to learn how to translate technical problems to customers (often bad news like expensive repairs) and tone choice can make the difference on that conversation being easy or not.

**Comment by edkranz** [comment]
People subconsciously respond to the type of tone you are using. If your aim for an interaction is to have a productive and confident discussion, then focusing your tone in this way will help. I've about 6 years experience working customer service, and I think that different tones are very important for conveying meaning and navigating situations. You don't always want to use a cheerful, confident tone. Context matters here.
also I think written tone is important too, and perhaps more difficult to master

**Comment by ricksu978** [comment]
Yes, tone + the choice of words could change how I was perceived.

**Comment by wicksipedia** [comment]
Years of retail sales has taught me that the expression your facial expression influences your tone. People can 'hear' a smile, even over a phone call. Smiling before speaking makes you sound more upbeat and engaged.

**Comment by Marxoz** [comment]
Yes and Yes :)

**Comment by steven0x51** [comment]
Yes, when I just started learning English my pitch was very high, and then I practiced to lower the pitch.

**Comment by isaaclombardssw** [comment]
I've tried, not always easy.

You can convince yourself that you're in a different mood to an extent, and that's usually what I find works best.

**Comment by bahjatalaadelssw** [comment]
Yes, I change tone mainly to clearly convey interest and intent.

**Comment by chrisschultzssw** [comment]
> In the past, have you tried to change your tone?
> Were your successful?

I haven't actively tried to change my tone, but I definitely speak differently in different contexts.
I should actively think about my tone in conversations with clients; I'm guilty of the upward inflexion thing?

**Comment by amankumarrr** [comment]
Yes, I’ve tried, but I find it challenging to think about both tone and sentence structure at the same time. However, I believe tone and word choice are really important. When I first came to Australia, I often used the word "no" in general conversations, which sometimes came across as unwillingness. Now, I’ve started using phrases like "I'm afraid not," which has the same meaning but softens the tone, making it clearer and more polite

**Comment by jaydenalchin** [comment]
Tone of voice is definitely important but so is facial expression & body language!

**Comment by ncn-ssw** [comment]
> In the past, have you tried to change your tone?

Yes - early in a career in another industry, I had found that people did not want to follow someone that did no sound confident and engaged.

> Were your successful?

Yes, and it had the desired effect.

**Comment by Ben0189** [comment]
Yes, challenging at start but getting better😊

**Comment by StefStar** [comment]
Yes - I had to change my tone for when I was working in hospitality as it's absolutely crucial in those environments. 

Yes I was successful, I wouldn't have made much of a career in hospitality if I had failed!

**Comment by zacharykeeping** [comment]
Yeah, depending on the context I might be more consciously aware of my tone and how I want to come across. I think I can usually be successful if I'm aware of it in these situations.

**Comment by Tanya-Leahy** [comment]
>In the past, have you tried to change your tone?

Yes - I have had to change my tone when working in my previous role. A lot of changes were being implemented and to gain confidence with the staff about these changes, I had to be mindful of my tone.

>Were your successful?

Yes, staff felt reassured and confident the changes were moving the organization towards a positive direction.

**Comment by PennyWalker** [comment]
Yes - in customer service you have to do this. You might be in a terrible mood, but you can;t let the customer know that so you have to smile (which instantly changes your tone!!), and then lift your voice to sell it. 

When something is super important I use tone to deliver it, so it's not lost in amongst the conversation.

**Comment by bradystroud** [comment]
> In the past, have you tried to change your tone?

No - but since filming the video yesterday its hard not to think about it

**Comment by miichaelsmedley** [comment]
> https://www.ssw.com.au/rules/match-tone-with-intent
> 
> In the past, have you tried to change your tone? Were your successful?

yes - I sometimes have too much tone, and there are times when the late-night DJ voice works better - calmness.

**Comment by jeoffreyfischer** [comment]
> In the past, have you tried to change your tone?

I've wanted to be never actively done it

> Were your successful?

No, but I will work on it now during the Scrum Meetings

**Comment by bettybondoc** [comment]
> In the past, have you tried to change your tone?

Yes, I was very shy and introverted as a child. I would often just stare at strangers who tried to talk to me. And when I do speak, I speak very softly and would always sound uncertain. It took a lot of conscious effort to become more outgoing, approachable, and to speak with confidence.

> Were your successful?

I believe it was a gradual but successful change. I also learned how to mask my annoyance with a smile when I started working. Now, no one believes me when I say I'm an introvert! 🤷🏻‍♀️

**Comment by Calinator444** [comment]
> In the past, have you tried to change your tone?

I can't say I have. I will need to focus on my tone of voice to prepare me for client work.

> Were your successful?

N/a

**Comment by danielmackay** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

I believe so, but it's something I'm conscious of and am constantly trying to improve.  It's not natural to me, as Kiwi's tend to be deadpan by nature.

> [!TIP]
> If you are motivated to work on your communication (as we should do, being Consultants), there are multiple opportunities daily that we can practice:
>  - talking to a neighbour
>  - talking to a cashier
>  - buying petrol at the servo

**Reply by SamBWagner** [reply]
Great tips!

**Comment by vladislav-kir** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

I think so, although I prefer following the flow instead and being natural.

**Comment by drwharris** [comment]
> In the past, have you tried to change your tone?
No, I am tone deaf. 

> Were your successful?
I am not sure, I think I do ok

**Comment by jimmidier** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

It was when I tried, but didn't keep on for long

**Comment by tino-liu** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

If i consciously think that I'll need to change my tone, then yes. And there are times when it triggers like a habit.

**Comment by Aibono1225** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

I think I did well at that time

**Comment by jerryluossw** [comment]
> In the past, have you tried to change your tone?

Yes, when I'm meeting with clients or presenting, I always try to sound energetic and confident.

> Were your successful?

I think it's good.

**Comment by sylhuang** [comment]
Yes - I found my tone sometimes aggressive and easily sounds like accusation rather than statement. Pretty sure I got this from my mom and it's easier to spot the issue watching her make the same mistake. When I'm consiously paying attension to this and I got a bit better. But the issue still comes up occasionally.

**Comment by babakamyljanovssw** [comment]
> In the past, have you tried to change your tone?

I don't recall whether I have tried to change my tone in the past.

> Were your successful?

N/A - I will be working on it during Daily Scrum meetings and improve my tone

**Comment by AnkyChe** [comment]
Yes, in the past, I spoke fast with an urgent tone, it made others stressed. Now, I speak slower, and others feel better when they talk to me. But I still tend to speak fast with an urgent tone when I'm in a hurry.

**Comment by jimmy1992abc** [comment]
> In the past, have you tried to change your tone?

Yes, I tried that when I spoke to juniors, I would speak more formally. However, when comes to seniors, I would speak more uncertainly to let them offer their opinions.

> Were you successful?

I don't know, but I think I should give others confidence when I express myself by using less uncertain tones.

**Comment by matt-goldman-ssw** [comment]
Continually adapting your tone to the audience and circumstances is one of the most effective communication tools there is (probably second only to empathy).

**Comment by duanxinhuan** [comment]
I know how to do it.

**Comment by AntPolkanov** [comment]
> In the past, have you tried to change your tone?
Were your successful?

Yeah, I used to sound monotonous. 
Hope I am getting better 😄

**Comment by MattParkerDev** [comment]
> In the past, have you tried to change your tone?

Not consciously.

> Were your successful?

🤷‍♂️

**Comment by stephanfakossw** [comment]
> In the past, have you tried to change your tone?
Yes - my student job was customer service in a call centre so tone really mattered

> Were your successful?
I think so - takes practice tho

**Comment by ozair-ashfaque** [comment]
> In the past, have you tried to change your tone?

Yes

> Were your successful?

Yes
